.gitignore

```diff
@@ -1,4 +1,6 @@
 Tests/Resources/app/cache
 Tests/Resources/app/logs
+bin/
 composer.lock
-vendor
+vendor/
+.idea
```

Comment:
this is the responsibility of the global git configuration of the developer, it is not specific to the project but to specific development environments. please remove this line.